modules/relevatehealthBidAdapter.js

```diff
@@ -0,0 +1,151 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { deepAccess, generateUUID, isArray, logError } from '../src/utils.js';
+
+const BIDDER_CODE = 'relevatehealth';
+
+const ENDPOINT_URL = 'https://rtb.relevate.health/prebid/relevate';
+
+function buildRequests(bidRequests, bidderRequest) {
+  const requests = [];
+  // Loop through each bid request
+  bidRequests.forEach(bid => {
+    // Construct the bid request object
+    const request = {
+      id: generateUUID(),
+      placementId: bid.params.placementId,
+      imp: [{
+        id: bid.bidId,
+        banner: getBanner(bid),
+        bidfloor: bid.params.bid_floor
```

Comment:
Please also look to the getFloor function in the request

Reply:
Created a function getFloor() to get bid_floor.